bot/discordbot.py

```diff
@@ -174,6 +178,94 @@ async def hello(interaction: discord.Interaction):
     await interaction.response.send_message(f"Hey {interaction.user.mention}! This is a slash command!")
 
 
+# Function for server admin to register credential info into Developer Console using '/register' command in the "get-verified" channel
+@bot.tree.command(name="register")
+async def register(interaction: discord.Interaction, email:str):
+
+    #Check if the command is used in the "get-verified" channel
+    channels = discord.utils.get(interaction.guild.channels, name='get-verified')
+    if not channels:
+        channels = await interaction.guild.create_text_channel('get-verified')
+    if interaction.channel.name != 'get-verified':
+        # message was not sent in the allowed channel
+        await interaction.response.send_message(
+            'This command can only be used in the <#{}> channel.'.format(str(channels.id)),
+            ephemeral=True
+        )
+        return
+    
+    # Check if the server already has a valid credential
+    server_id = interaction.guild_id
+    BACKEND_URL = env["DISCORD_BACKEND_URL"]
+    response = requests.get(
+        BACKEND_URL + '/api/?serverId=' + str(server_id)
+    )
+    if response.status_code != 400:
+        await interaction.response.send_message(
+            "The server already has have a valid credential.",
```

Comment:
Looks like you have a typo here 😄  
The server already has ~have~ a valid credential